src/DeprecateKeywords.jl

```diff
@@ -65,10 +65,12 @@ function _depkws(def)
 
     # Update new symbols to use deprecated kws if passed:
     for (i, kw) in enumerate(sdef[:kwargs])
-        isa(kw, Symbol) && continue
-
-        new_kw = kw.args[1]
-        default = kw.args[2]
+        no_default = isa(kw, Symbol)
```

Comment:
Actually this does not account for things like `; a::Int`. Needs to be more general.

Reply:
Fixed.